data/xml/2024.argmining.xml

```diff
@@ -15,6 +15,10 @@
       <url hash="fd001b46">2024.argmining-1</url>
       <venue>argmining</venue>
     </meta>
+    <frontmatter>
+      <url hash="afec1ab1">2024.argmining-1.0</url>
+      <bibkey>2024.argmining-1</bibkey>
```

Comment:
This should probably be something like `argmining-2024-1` or `argmining-2024-proceedings`.